### DISCUSSION TITLE
Simulating NAT, making a node inaccessible

### DISCUSSION THREAD

**Discussion by reedrosenbluth** [testground/testground#1196] [Q&A]
Hi! Is it possible to make a node inaccessible to the rest of the nodes in the network, but maintain its outward connection to them?

**Comment by raulk** [comment]
@reedrosenbluth Hey! Welcome to the Testground community ;-)

Yes, you should be able to set the `Drop` `FilterAction` in the [`LinkShape`](https://pkg.go.dev/github.com/testground/sdk-go@v0.2.7/network#LinkShape) when you request a network configuration change via the SDK's `network.Client#ConfigureNetwork()` method.

You can check out the libp2p/ping test plan for a walkthrough on how to set network traffic shaping policies: https://github.com/libp2p/test-plans/tree/master/ping

Hope that helps!

**Reply by reedrosenbluth** [reply]
Cool, thanks!
